Remove BBC radio player quirk leftovers

blino opened the pull request:
#### e0366ad8cf403e21b813ddd94bc2441d27c60fa4
<pre>
Remove BBC radio player quirk leftovers
<a href="https://bugs.webkit.org/show_bug.cgi?id=262300">https://bugs.webkit.org/show_bug.cgi?id=262300</a>

Reviewed by Kate Cheney and Brent Fulgham.

The BBC radio player quirk has been removed in 266597@main (9b58b4b),
which is a back-merge of 259548.824@safari-7615-branch (cff01e3).

But this commit did not remove the Quirks::staticRadioPlayerURLString() method
initially removed, and a second quirk part added in 261190@main (ae69009)
which is now irrelevant and potentially broken.

The usage of staticRadioPlayerURLString() also broke build without ITP support.

* Source/WebCore/page/Quirks.cpp:
(WebCore::Quirks::staticRadioPlayerURLString): Deleted.
* Source/WebCore/page/Quirks.h:
* Source/WebKit/UIProcess/WebPageProxy.cpp:
(WebKit::WebPageProxy::createNewPage):

Canonical link: <a href="https://commits.webkit.org/268769@main">https://commits.webkit.org/268769@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/e334db5a6bcb92ebc2ff352a924f876bf7ae9f4b

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/20089 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/20528 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/21146 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/21988 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/18753 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/20319 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/23770 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/20686 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/20229 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/20309 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/20226 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/17459 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/22837 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/17396 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/18256 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/24509 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/18473 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/18432 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/22499 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/19029 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/16155 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/18225 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/4950 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/22569 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/18854 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

brentfulgham reviewed (approved):
Agreed -- we missed this bit of the old Quirk when we removed the rest. Thank you for catching this!

blino committed:
Remove BBC radio player quirk leftovers
https://bugs.webkit.org/show_bug.cgi?id=262300

Reviewed by Kate Cheney and Brent Fulgham.

The BBC radio player quirk has been removed in 266597@main (9b58b4b),
which is a back-merge of 259548.824@safari-7615-branch (cff01e3).

But this commit did not remove the Quirks::staticRadioPlayerURLString() method
initially removed, and a second quirk part added in 261190@main (ae69009)
which is now irrelevant and potentially broken.

The usage of staticRadioPlayerURLString() also broke build without ITP support.

* Source/WebCore/page/Quirks.cpp:
(WebCore::Quirks::staticRadioPlayerURLString): Deleted.
* Source/WebCore/page/Quirks.h:
* Source/WebKit/UIProcess/WebPageProxy.cpp:
(WebKit::WebPageProxy::createNewPage):

Canonical link: https://commits.webkit.org/268769@main
